app/bindings/api/v0/bindings/presentation.rb

```diff
@@ -0,0 +1,234 @@
+=begin
+#Assessment Network Interactions API
+
+#Records interactions with content distributed by the Assessment Network.  Requests to this API should include `application/json` in the `Accept` header.  The desired API version is specified in the request URL, e.g. `...a15k.org/v0/flags`.          While the API does support a default version, that version will change over          time and therefore should not be used in production code!  Some endpoints require an API key to be passed in the request header.  There are two          types of API keys: API tokens and API IDs.  An API token is used for more restricted          access.  Such tokens should not be shared with end users.  API IDs are used for less          restricted access and may be exposed to clients (e.g. through use in browser-side code).          Both keys are available to members through www.a15k.org. 
+
```

Comment:
what's up with the weird spacing here?  When I viewed the file by itself all this is on one line, maybe you somehow are using windows line breaks or something?

Reply:
oh nm, I see this is auto-generated

Reply:
yeah, I added `\` in the markdown string so it renders nicely on swagger-ui... unfortunately I guess that means it renders terribly here.